reconcile/cli.py

```diff
@@ -1049,7 +1049,8 @@ def terraform_resources(ctx, print_only, enable_deletion,
                     ctx.obj, print_only,
                     enable_deletion, io_dir, thread_pool_size,
                     internal, use_jump_host, light, vault_output_path,
-                    account_name=account_name)
+                    account_name=account_name,
+                    extra_labels=ctx.obj['extra_labels'])
```

Comment:
Just one doubt. If we run the integration manually, is this set to `None` by some Click internal magic?

Reply:
i'll take another look at this

Reply:
you are correct. to make things simpler, i changed this to `ctx.obj.get('extra_labels', {})`, as all we're doing inside the integration is setting a value on a dict.